lizmap/www/assets/js/map.js

```diff
@@ -3975,20 +4039,56 @@ var lizMap = function() {
       }
 
       // if user has made a visible draw, print it with redlining
+      const formatWKT = new OpenLayers.Format.WKT();
+      const highlightGeom = [];
+      const highlightSymbol = [];
       if (lizMap.mainLizmap.digitizing.featureDrawn && lizMap.mainLizmap.digitizing.featureDrawnVisibility){
-        const formatWKT = new OpenLayers.Format.WKT();
-
-        const highlightGeom = [];
-        const highlightSymbol = [];
-
         for (let index = 0; index < lizMap.mainLizmap.digitizing.featureDrawn.length; index++) {
-          highlightGeom.push(formatWKT.write(lizMap.mainLizmap.digitizing.featureDrawn[index]));
+          var draw_feature = lizMap.mainLizmap.digitizing.featureDrawn[index];
+          if (printInProjectProjection
+            && lizMap.config.options.qgisProjectProjection.ref != lizMap.config.options.projection.ref
+          ) {
+            // If we print in the QGIS project projection
+            draw_feature.geometry.transform(map.projection, project_projection);
+          }
+          highlightGeom.push(formatWKT.write(draw_feature));
           highlightSymbol.push(lizMap.mainLizmap.digitizing.getFeatureDrawnSLD(index));
 
         }
+      }
+      // Print the chosen drawn rectangle in dashed pale white
+      var printDrawBox = false;
+      if ('printDrawBox' in lizMap.config.options && lizMap.config.options.printDrawBox == 'True') {
```

Comment:
Where `lizMap.config.options.printDrawBox` is set?

Reply:
It was added there so that an external JS script could set this property for debugging purpose. Not sure if we need to keep it ? If not, we need to remove the corresponding code @rldhont any thought ?